TestLibrary/ControllersInstallerTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Linq;
+using System.Web.Mvc;
+using Castle.Core;
+using Castle.Core.Internal;
+using Castle.MicroKernel;
+using Castle.Windsor;
+using Some.Controllers;
+using Some.Installers;
+using Xunit;
+
+namespace TestLibrary
+{
+    public class ControllersInstallerTests
+    {
+        private IWindsorContainer contanierWithControllers;
+
+        public ControllersInstallerTests()
+        {
+            contanierWithControllers = new WindsorContainer()
+                .Install(new ControllerInstaller());
+        }
+
+        [Fact]
+        public void All_controllers_implement_IController()
```

Comment:
Usually better to name test method with this pattarn:
ActionWichWeAreTesting_Preconditions_ExpectedResult